packages/auth/src/core/strategies/email_and_password.ts

```diff
@@ -74,16 +78,43 @@ export async function sendPasswordResetEmail(
   email: string,
   actionCodeSettings?: ActionCodeSettings
 ): Promise<void> {
-  const authModular = getModularInstance(auth);
-  const request: authentication.PasswordResetRequest = {
-    requestType: ActionCodeOperation.PASSWORD_RESET,
-    email
-  };
-  if (actionCodeSettings) {
-    _setActionCodeSettingsOnRequest(authModular, request, actionCodeSettings);
+  const authInternal = _castAuth(auth);
+
+  async function internalSendPasswordResetEmail(withRecaptcha = false): Promise<PasswordResetResponse> {
+    const request: authentication.PasswordResetRequest = {
+      requestType: ActionCodeOperation.PASSWORD_RESET,
+      email
+    };
+    if (withRecaptcha) {
+      const verifier = new RecaptchaEnterpriseVerifier(auth);
+      const captchaResponse = await verifier.verify('passwordReset')
+      .catch(() => {
```

Comment:
Please convert this to a try/catch as well

Reply:
I was planning to update these with the generic helper method that we discussed above. Convert to try/catch for now.